terraform/variables.tf

```
@@ -274,7 +274,7 @@ variable "findings_data_save_succeeded" {
 
 variable "findings_data_import_lambda_s3_bucket" {
   type        = string
-  description = "The name of the bucket where the findings data import Lambda function can be found.  This bucket should be created with https://github.com/cisagov/findings-data-import-terraform.  Note that in production terraform workspaces, the string '-production' will be appended to the bucket name.  In non-production workspaces, '-<workspace_name>' will be appended to the bucket name."
+  description = "The name of the bucket where the findings data import Lambda function can be found.  This bucket should be created with the cisagov/findings-data-import-terraform project.  Note that in production terraform workspaces, the string '-production' will be appended to the bucket name.  In non-production workspaces, '-<workspace_name>' will be appended to the bucket name."
```

Keep the capitalization train rollin':
```suggestion
  description = "The name of the bucket where the findings data import Lambda function can be found.  This bucket should be created with the cisagov/findings-data-import-terraform project.  Note that in production Terraform workspaces, the string '-production' will be appended to the bucket name.  In non-production workspaces, '-<workspace_name>' will be appended to the bucket name."
```

---

Please see [b1bf5f0](https://github.com/cisagov/cyhy_amis/pull/361/commits/b1bf5f0a3dd5f7ff6e973ec2c15582f4a3cd2b56).